Community/Tdarr_Plugin_goof1_URL_Plex_Refresh.js

```diff
@@ -0,0 +1,221 @@
+              The Library key is pointed out here as an example. 
+              This number will be used here for the library relevant to the content being transcoded within this TDarr library. 
+
+              *Note* If this number is wrong everything will behave as though it's working great but the folder will simply not be scanned. 
+
+              https://support.plex.tv/articles/201638786-plex-media-server-url-commands/
+
+               \\nExample:\\n
+               29`,
+    },
+    {
+      name: 'Plex_Path',
+      type: 'string',
+      defaultValue: '',
+      inputUI: {
+        type: 'text',
+      },
+      tooltip: `
+              If the Plex path is not the same as the local path you may need to sub parts of the path. 
+              Here is where you would enter the path that Plex uses to find the file. You would only enter the part of the path that is different.
+
+              If your TDarr path is:
+              /media/local/tv/The Best Show Evaaaarr/Season 2/The Best Show Evaaaarr - S02E31 - Heck Yea HDTV-720p.mp4
+              
+              And the Plex path to the file is:
+              /data/tv/The Best Show Evaaaarr/Season 2/The Best Show Evaaaarr - S02E31 - Heck Yea HDTV-720p.mp4
+
+              then part you would enter here is:
+
+               \\nExample:\\n
+               /data/`,
+    },
+    {
+      name: 'TDarr_Path',
+      type: 'string',
+      defaultValue: '',
+      inputUI: {
+        type: 'text',
+      },
+      tooltip: `
+              If the Plex path is not the same as the local path you may need to sub parts of the path. 
+              Here is where you would enter the path that Plex uses to find the file. You would only enter the part of the path that is different.
+
+              If your TDarr path is:
+              /media/local/tv/The Best Show Evaaaarr/Season 2/The Best Show Evaaaarr - S02E31 - Heck Yea HDTV-720p.mp4
+              
+              And the Plex path to the file is:
+              /data/tv/The Best Show Evaaaarr/Season 2/The Best Show Evaaaarr - S02E31 - Heck Yea HDTV-720p.mp4
+
+              then part you would enter here is:
+
+               \\nExample:\\n
+               /media/local/`,
+    },
+  ],
+});
+
+const plugin = (file, librarySettings, inputs, otherArguments) => {
+  inputs = loadDefaultValues(inputs, details);
+
+  let response = {
+    file,
+    removeFromDB: false,
+    updateDB: false,
+    infoLog: '',
+  };
+
+  const http = require('http')
+  const https = require('https')
+
+  const type = inputs.Url_Protocol
+  const url = inputs.Plex_Url
+  const port = inputs.Plex_Port
+  const token = inputs.Plex_Token
+  const key = inputs.Library_Key
+  const plexPath = inputs.Plex_Path
+  const tdarrPath = inputs.TDarr_Path
+
+  if (!type || !url || !token || !key) {
+    response.infoLog = `Url_Protocol, Plex_Url, Plex_Token, and Library_Key are all required`
+    response.processFile = false
```

Comment:
You can set `processFile` to `false` by default by having it in the response object when you first create it